src/resolve-script-name.js

```diff
@@ -0,0 +1,28 @@
+import {get} from 'lodash'
+
+export default resolveScriptName
+
+function resolveScriptName(scriptToFind, scriptsConfig) {
```

Comment:
> Couldn't we extend getScriptToRun to get the script name in addition to the script command?

Maybe. Without looking at the code I'm unsure. If so, that'd be great.

> reimplementing `prefix-matcher`

Maybe we could contribute back and expose a method to do what we need :)